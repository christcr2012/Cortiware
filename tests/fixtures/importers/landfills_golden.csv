id,name,lat,lon,accepts
LF-TEST1,Test North,40.0,-105.5,"msw;c&d"
LF-TEST2,Test South,39.5,-105.0,msw

