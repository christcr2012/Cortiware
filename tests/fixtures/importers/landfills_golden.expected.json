[
  {
    "id": "LF-TEST1",
    "name": "Test North",
    "lat": 40,
    "lon": -105.5,
    "accepts": ["msw", "c&d"]
  },
  {
    "id": "LF-TEST2",
    "name": "Test South",
    "lat": 39.5,
    "lon": -105,
    "accepts": ["msw"]
  }
]

